dotcom-rendering/src/web/components/Border.tsx

```diff
@@ -1,14 +1,26 @@
 import { css } from '@emotion/react';
+import { ArticleSpecial } from '@guardian/libs';
 import { from } from '@guardian/source-foundations';
 import { decidePalette } from '../lib/decidePalette';
 
-export const Border = ({ format }: { format: ArticleFormat }) => (
-	<div
-		css={css`
+const borderStyles = (format: ArticleFormat) => {
+	if (format.theme === ArticleSpecial.SpecialReportAlt) {
+		return css`
 			${from.leftCol} {
-				border-left: 1px solid ${decidePalette(format).border.article};
+				border-left: 1px solid rgba(60, 60, 60, 0.3);
```

Comment:
Is there another way of doing this in DCR?

Reply:
Used [`transparentColour`](https://github.com/guardian/dotcom-rendering/blob/3ac24426c459fb8456c81b5148da6c883872189e/dotcom-rendering/src/web/lib/transparentColour.ts#L26)